readthedocs/search/tests/test_utils.py

```diff
@@ -0,0 +1,77 @@
+import time
+
+import pytest
+from django.urls import reverse
+
+from readthedocs.builds.constants import LATEST, STABLE
+from readthedocs.projects.models import HTMLFile
+from readthedocs.search import utils
+from readthedocs.search.tests.utils import get_search_query_from_project_file
+
+
+@pytest.mark.django_db
+@pytest.mark.search
+@pytest.mark.proxito
+class TestSearchUtils:
```

Comment:
This could use a docstring.

Reply:
Not sure what to put in a test class, methods are named according to the test case.